concourse/model/traits/publish.py

```diff
@@ -78,6 +82,11 @@
         default=None,
         doc='only for multistage builds: the target up to which to build',
     ),
+    AttributeSpec.optional(
+        name='component_descriptor_labels',
+        default=[],
```

Comment:
is there an issue if we use the empty tuple instead of the empty list as default value? in general, default values should preferrably be immutable..